qiskit_ibm_runtime/options/environment_options.py

```diff
@@ -52,12 +52,14 @@ class EnvironmentOptions:
         Default: ``None``.
     """
     private: Optional[bool] = False
-    r"""Boolean that indicates whether the job is marked as private. This is only
-        supported for ``ibm_quantum`` channel. When set to true, input parameters are not
-        returned, and the results can only be read once. After the results are read or after
-        a specified time after the job is completed, the results are deleted from the service.
-        When set to false, the input parameters and results follow the standard retention
-        behavior.
+    r"""Boolean that indicates whether the job is marked as private. When set to true, 
+        input parameters are not returned, and the results can only be read once. 
+        After the job is completed, input parameters are deleted from the service. 
+        After the results are read, they are deleted from the service. 
+        When set to false, the input parameters and results follow the 
+        standard retention behavior of the API.
+        Only returned in the job response if the value is true, 
+        otherwise it is omitted.
```

Comment:
Is this what the last sentence meant? I am not sure what is "only returned", the value of the flag?

```suggestion
        When set to false, the input parameters and results follow the 
        standard retention behavior of the API.
        This parameter is only returned in the job response if the value is true, 
        otherwise it is omitted.
```

Reply:
I actually tested the PR and I don't think it's the parameter being returned:

`PrimitiveResult([SamplerPubResult(data=DataBin(c=BitArray(<shape=(), num_shots=4096, num_bits=4>)), metadata={'circuit_metadata': {}})], metadata={'execution': {'execution_spans': ExecutionSpans([DoubleSliceSpan(<start='2025-05-27 15:55:46', stop='2025-05-27 15:55:46', size=4096>)])}, 'version': 2})`

Reply:
Clarified the docs and removed this last sentence in the latest commit since the wording is confusing/misleading. The `private` attribute is not returned in the api job response when it is not set but from the provider, `job.private` will be false.